Snippets/Core/Core_6/Pipeline/OnReceivePipelineCompleted.cs

```diff
@@ -0,0 +1,25 @@
+namespace Core6.Pipeline
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    class OnReceivePipelineCompleted
+    {
+        public OnReceivePipelineCompleted()
+        {
+            var endpointConfiguration = new EndpointConfiguration("MyEndpoint");
+
+            #region ReceivePipelineCompletedSubscription
+
+            endpointConfiguration.Pipeline.OnReceivePipelineCompleted(e =>
```

Comment:
When writing a feature, you would use `context.Pipeline.OnReceivePipelineCompleted(e =>`. Would that not be a better example to provide?